docs/getting-started/community-membership.sh

```diff
@@ -0,0 +1,119 @@
+:# KubeEdge Community membership
+
+**Note :** This document keeps changing based on the status and feedback of KubeEdge Community.
+
+This document gives a brief overview of the KubeEdge community roles with the requirements and responsibilities associated with them.
+
+| Role | Requirements | Responsibilities | Privileges |
+| -----| ---------------- | ------------ | -------|
+| [Member](#member) | Sponsor from 2 approvers, active in community, contributed to KubeEdge | Welcome and guide new contributors | KubeEdge GitHub organization Member |
```

Comment:
Currently we have the collaborator role, does it mean we should transfer all the collaborators to new members?

Reply:
I think so, github repo collaborator only works for indicated repo, and expensive to manage since there's no batch management operation.